docs/configuration_file.md

```diff
@@ -59,6 +59,22 @@ Some options are also mandatory.
 
 * `koji_kerberos_principal` (*optional*, `string`) - kerberos principal for the keytab provided in `koji_kerberos_keytab`
 
+* `flatpak` (*optional*, `boolean`) - If true, will build a Flatpak as an OCI image rather than a docker image
+
+* `module` (*optional*, `string`) - Module to build the image against, in the form `NAME:STREAM[:VERSION]`. Currently used only for flatpaks.
```

Comment:
Yeah - while the way that the code works most things can be configured (even target/git_uri, etc.) these don't make much sense as configuration parameters and shouldn't be documented here.